scripts/upgrade

```diff
@@ -241,16 +241,17 @@ then
 	# Load the last available version
 	source ./upgrade.d/upgrade.last.sh
 	last_version=$gitlab_version
-	last_major_version=${gitlab_version%%.*}
 
 	# While the current version is not the last version, do an upgrade
 	while [ "$last_version" != "$current_version" ]
 	do
 
 		current_major_version=${current_version%%.*}
 
-		# If the current version is equal to the last minor version of upgrade.$current_major_version.sh, increment the major version
-		if [ "$gitlab_version" = "$current_version" ]; then
+		source ./upgrade.d/upgrade.$current_major_version.sh
+
+		# f the current version is higher than the version stored in upgrade.$current_major_version.sh, increment the major version
+		if dpkg --compare-versions "$gitlab_version" "le" "$current_version"; then
```

Comment:
Not 100% sure of using "le"... maybe better to use "gt"
I've some difficulties to decrypt what is going on...
Just remember that the result of "dpkg --compare-versions" return code is 0 if the sentence is true and the return code is not null if the sentence is false

Reply:
Here `gitlab_version` is the version of the file `./upgrade.d/upgrade.$current_major_version.sh`. If this version is less than or equal to `current_version`, then this update doesn't have to be done, we can go ahead :-)

Reply:
Thank you for your review!